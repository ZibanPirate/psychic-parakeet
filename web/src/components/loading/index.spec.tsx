import { cleanup, render } from "@testing-library/react";
import { Loading } from ".";

describe("Loading Component", () => {
  afterEach(() => cleanup());

  test("Render Loading component", () => {
    const renderedLoading = render(<Loading />);
    expect(renderedLoading.container).toMatchSnapshot();
  });
});
