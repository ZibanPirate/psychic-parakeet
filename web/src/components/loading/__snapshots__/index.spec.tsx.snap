// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Render Loading component 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <img
      alt="parakeet"
      class="makeStyles-image-2"
      src="test-file-stub"
    />
  </div>
</div>
`;
